wp-content/civi-extensions/goonjcustom/Civi/MaterialContributionService.php

```diff
@@ -104,6 +109,11 @@ public static function attachContributionReceiptToEmail(&$params, $context = NUL
 
     $locationAreaOfCamp = $collectionCamp['Collection_Camp_Intent_Details.Location_Area_of_camp'] ?? 'N/A';
 
+    // If locationAreaOfCamp is 'N/A' and city is not empty, assign the value of city to locationAreaOfCamp
+    if ($locationAreaOfCamp === 'N/A' && !empty($city)) {
```

Comment:
How does the value `N/A` gets stored in $locationAreaOfCamp?

Reply:
If Location_Area_of_camp is not present in the `$collectionCamp['Collection_Camp_Intent_Details.Location_Area_of_camp']`  or if its value is null, then 'N/A' is used as a fallback.

I have removed this line after refactoring